docs/content/howto/logging/send_table.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Send tables to Rerun
+order: 0
+description: Shows how to send tables as dataframes to the Rerun viewer.
+---
+
+> **Note:** The `send_table` API is currently experimental and may change in future releases.
+
+Rerun now supports sending tabular data to the Rerun Viewer! This feature allows you to visualize and interact with dataframes (encoded as Arrow record batches) directly in the Rerun Viewer environment.
+
+## Overview
+
+The `send_table` API provides a straightforward way to send tabular data to the Rerun Viewer. This is particularly useful for:
+
+- Inspecting dataframes alongside other visualizations
+- Debugging data processing pipelines
+- Presenting structured data in a readable format
+
+## References
+
+For complete examples of using `send_table`, please refer to:
+
+- [🐍 Jupyter Notebook](https://github.com/rerun-io/rerun/blob/main/examples/python/notebook/send_table.ipynb)
+- [🐍 Python SDK](https://github.com/rerun-io/rerun/blob/main/docs/snippets/all/howto/send_table.py)
+
+## Prerequisites
+
+- Rerun SDK (Python)
+- PyArrow library
+- Pandas
+- NumPy
+
+Which can be installed via:
+
+```sh
+pip install rerun-sdk[notebook] pyarrow pandas numpy
+```
+
+## Basic usage
+
+### Connecting to the Viewer
+
+```python
+from rerun_bindings import ViewerClient
+
+# Connect to a running Rerun Viewer
+client = ViewerClient(addr="rerun+http://0.0.0.0:9876")
+```
+
+### Sending a simple table
+
+```python
+import pyarrow as pa
+
+# Create a record batch from a Python dictionary
+record_batch = pa.RecordBatch.from_pydict({
+    "id": [1, 2, 3],
+    "url": ["https://www.rerun.io", "https://github.com/rerun-io/rerun", "https://crates.io/crates/rerun"],
+})
+
+# Send the table to the viewer with an identifier
+client.send_table("My Table", record_batch)
+```
+
+### Using with Pandas dataframes
+
+You can also send Pandas dataframes by converting them to a record batch:
+
+```python
+import pandas as pd
+import numpy as np
+import pyarrow as pa
+
+# Create a sample DataFrame
+dates = pd.date_range("20230101", periods=6)
+df = pd.DataFrame(np.random.randn(6, 4), index=dates, columns=list("ABCD"))
+
+# Convert to record batch and send to the viewer.
+client.send_table("Pandas DataFrame", pa.RecordBatch.from_pandas(df))
```

Comment:
Could we do this `pa.RecordBatch.from_pandas` call in `send_table` instead (so the user doesn't have to)? If not, why not? Is it just that we don't want to depend on pandas?